compiler/damlc/base-rst-index-template.rst

```diff
@@ -8,7 +8,7 @@ The standard library
 
 The Daml standard library is a collection of Daml modules that are bundled with the SDK, and can be used to implement Daml applications.
 
-The :ref:`Prelude <module-prelude-6842>` module is imported automatically in every Daml module. Other modules must be imported manually, just like your own project's modules. For example:
+The :ref:`Prelude <module-prelude-72703>` module is imported automatically in every Daml module. Other modules must be imported manually, just like your own project's modules. For example:
```

Comment:
how did you figure out the new ref numbers?

Reply:
Stare at what we generate and adjust. There is no great way unfortunately. The reason it changed is because the Hashable instance for Text changed. I discussed it with @sofiafaro-da and we agreed to accept this for now but we opened https://github.com/digital-asset/daml/issues/12612 to address this in the future.

Reply:
oh right I saw that message. thanks for opening the issue too